script.js

```diff
@@ -1,18 +1,50 @@
 var scissors = "scissors";
 var paper = "paper";
 var stone = "stone";
+var reversedScissors = "reversed scissors";
+var reversedPaper = "reversed paper";
+var reversedStone = "reversed stone";
+
+var randomNumber = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var randNumber = randomInteger;
+  console.log(randNumber);
+  return randNumber;
+};
+
+var computerChoice = function () {
+  var number = randomNumber();
+  var choice = scissors;
+
+  if (number == 0) {
+    choice = stone;
+  }
+
+  if (number == 1) {
+    choice = paper;
+  }
+  return choice;
+};
 
 var main = function (input) {
-  var player = input;
   var computer = computerChoice();
+  var player = input;
   console.log("Computer Choice");
   console.log(computer);
   console.log("User Choice");
   console.log(input);
```

Comment:
[Comment]: you can also combine this console logs into one line if you think it is cleaner
console.log ("User choice", input)